suzieq/engines/pandas/engineobj.py

```diff
@@ -201,7 +202,10 @@ def get_valid_df(self, **kwargs) -> pd.DataFrame:
         getcols = list(set(fields+addnl_fields))
 
         tz = self.ctxt.cfg.get('analyzer', {}).get('timezone', 'UTC')
-        settings = {'TIMEZONE': tz, 'TO_TIMEZONE': 'UTC'}
+        if tz not in all_timezones:
+            raise AttributeError(f'Invalid Timezone {tz} specified in config')
+
+        settings = {'TIMEZONE': tz}
```

Comment:
Isn't it better to have this check during the load of the sq_confing in the sqobject rather than in the engine?

Reply:
Its already there. Removed it. I found it crasjed on a mistake. Maybe its something else. need to validate